samples/matter/temperature_sensor/sample.yaml

```diff
@@ -0,0 +1,57 @@
+sample:
+  description: Matter Temperature Sensor example
+  name: Matter Temperature Sensor
+tests:
+  sample.matter.temperature_sensor.debug:
+    sysbuild: true
+    build_only: true
+    integration_platforms:
+      - nrf52840dk/nrf52840
+      - nrf5340dk/nrf5340/cpuapp
+      - nrf54l15dk/nrf54l15/cpuapp
+      - nrf54lm20dk/nrf54lm20a/cpuapp
+    platform_allow:
+      - nrf52840dk/nrf52840
+      - nrf5340dk/nrf5340/cpuapp
+      - nrf54l15dk/nrf54l15/cpuapp
+      - nrf54lm20dk/nrf54lm20a/cpuapp
+    tags:
+      - sysbuild
+      - ci_samples_matter
+  sample.matter.temperature_sensor.release:
```

Comment:
suggest to add to quarantine integration all release configurations to limit building time on PRs